user_docs/en/changes.md

```diff
@@ -5,6 +5,9 @@
 ### Important notes
 
 ### New Features
+* The volume of other applications can be adjusted by `NVDA+alt+pageUp` and `NVDA+alt+pageDown`. In order to use this feature, application volume adjuster needs to be enabled in Audio pane of NVDA settings. (#16052, @mltony)
+* Added command to mute or unmute all other applications, assigned to `NVDA+alt+delete`.
+In order to use this feature, the application volume adjuster needs to be enabled in Audio panel of NVDA settings. (#16052, @mltony)
```

Comment:
A detail, but the user rather speak about "category" when the developer speaks about a "panel".

```suggestion
In order to use this feature, the application volume adjuster needs to be enabled in Audio cetegory of NVDA settings. (#16052, @mltony)
```